scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -189,18 +148,6 @@ def call(final pipelineContext) {
       stageName: 'Java 8 XGBoost Multinode JUnit', target: 'test-junit-xgb-multi-jenkins', pythonVersion: '2.7', javaVersion: 8,
       timeoutValue: 120, component: pipelineContext.getBuildConfig().COMPONENT_JAVA, additionalTestPackages: [pipelineContext.getBuildConfig().COMPONENT_PY]
     ],
-    [
```

Comment:
Just run for the "enterprise" version of Java - 8

rest of it can be run in nightly